docs/user_guides/projects/airflow/airflow.md

```diff
@@ -0,0 +1,99 @@
+
+You can then create the DAG and Hopsworks will generate the Python file. 
+
+#### Write your own DAG 
+
+If you prefer to code the DAGs or you want to edit a DAG built with the builder tool, you can do so. The Airflow DAGs are stored in the _Airflow_ dataset which you can access using the file browser in the project settings.
+
+When writing the code for the DAG you can invoke the operator as follow:
+
+```python
+HopsworksLaunchOperator(dag=dag,
+	task_id="profiles_fg_0",
+	project_name="airflow_doc",
+	job_name="profiles_fg",
+	job_arguments="",
+	wait_for_completion=True)
+```
+
+You should provide the name of the Airflow task (`task_id`) and the Hopsworks job information (`project_name`, `job_name`, `job_argument`). You can set the `wait_for_completion` flag to `True` if you want the operator to block and wait for the job execution to be finished.
+
+Similarly, you can invoke the sensor as shown below. You should provide the name of the Airflow task (`task_id`) and the Hopsworks job information (`project_name`, `job_name`) 
+
+```python
+HopsworksJobSuccessSensor(dag=dag,
+    task_id='wait_for_profiles_fg',
+	project_name="airflow_doc",
+    job_name='profiles_fg')
+```
+
+When writing the DAG file, you should also add the `access_control` parameter to the DAG configuration. The `access_control` parameter specicifies which projects have access to the DAG and which actions the project members can perform on it. If you do not specify the `access_control` option, project members will not be able to see the DAG in the Airflow UI.
+
+!!! warning "Admin access"
+    The `access_control` configuration does not apply to Hopsworks admin users which have full access to all the DAGs even if they are not member of the project. 
+
+```python
+    dag = DAG(
+        dag_id = "example_dag",
+        default_args = args,
+        access_control = {
+            "project_name": {"can_dag_read", "can_dag_edit"},
+        },
+
+        schedule_interval = "0 4 * * *"
+    )
+```
+
+!!! note "Project Name"
+    You should replace the `project_name` in the snippet above with the name of your own project
+
+#### Manage Airflow DAGs using Git
+
+You can leverage the [Git integration](../git/clone_repo.md) to track your Airflow DAGs in a git repository. Airflow will only consider the DAG files which are stored in the _Airflow_ Dataset in Hopsworks. 
+After cloning the git repository in Hopsworks, you can automate the process of copying the DAG file in the _Airflow_ Dataset using the [copy method](https://docs.hopsworks.ai/hopsworks-api/{{{ hopsworks_version }}}/generated/api/datasets/#copy) of the Hopsworks API.
```

Comment:
Can you not just clone the git repo directly into the Airflow dataset? Why do you need to copy it to the Airflow dataset?

Reply:
Alt text

If you clone your git repository containing the Airflow DAGs in a directory that is not a subdirectory of _Airflow_, you can automate the process of copying the DAG file in the _Airflow_ Dataset using the [copy method](https://docs.hopsworks.ai/hopsworks-api/{{{ hopsworks_version }}}/generated/api/datasets/#copy) of the Hopsworks API.

Reply:
I don't think it's a good idea. Airflow will try to execute all the Python files you have in the repository and load the resulting dag.